src/buildCommon.js

```diff
@@ -146,16 +145,46 @@ const mathit = function(
     if (/[0-9]/.test(value.charAt(0)) ||
             // glyphs for \imath and \jmath do not exist in Math-Italic so we
             // need to use Main-Italic instead
-            utils.contains(mainitLetters, value)) {
+            utils.contains(mathitLetters, value)) {
         return {
             fontName: "Main-Italic",
-            fontClass: "mainit",
+            fontClass: "mathit",
         };
     } else {
         return {
             fontName: "Math-Italic",
+            fontClass: "mathdefault",
+        };
+    }
+};
+
+/**
+ * Determines which of the font names (Main-Italic, Math-Italic, and Caligraphic)
+ * and corresponding style tags (mathit, mathdefault, or mathcal) to use for font
+ * "mathnormal", depending on the symbol.  Use this function instead of fontMap for
+ * font "mathnormal".
+ */
+const mathnormal = function(
+    value: string,
+    mode: Mode,
+    options: Options,
+    classes: string[],
+): {| fontName: string, fontClass: string |} {
+    if (utils.contains(mathitLetters, value)) {
+        return {
+            fontName: "Main-Italic",
             fontClass: "mathit",
         };
+    } else if (/[0-9]/.test(value.charAt(0))) {
```

Comment:
It'd be nice to have some helper functions for some of these tests, maybe `isLatinLetter`, `isNumber`, etc.

Reply:
Even better (faster) would be to use `hasOwnProperty` on an object with the appropriate keys. We should replace all/most uses of `utils.contains` with this. But not necessarily for this PR.

Reply:
That's clever.  We'd want to construct the object at load time though.